.circleci/config.yml

```diff
@@ -42,6 +42,8 @@ commands:
           name:  "Setup rust environment"
           command: |
             curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain $(< rust-toolchain) && source $HOME/.cargo/env
+            rustup install nightly-2019-04-30
```

Comment:
🔧 anyway you can put the version in a variable so that we don't miss one when updating?

Reply:
You can't really miss one because the build fails if they are out of sync.